src/utils.js

```diff
@@ -873,43 +873,34 @@ export function toIdSchema(
 
 export function toPathSchema(schema, name = "", definitions, formData = {}) {
   const pathSchema = {
-    $name: name,
+    $name: name.replace(/\.$/, ""),
```

Comment:
Why is this needed?

Reply:
As a result of refactoring and because the initial call starts with name as an empty string, I ended up with a period prepended to all of the `$name`s, so I swapped it such that the period is appended, then removed when the `$name` value is set. e.g. here

https://github.com/Tonexus/react-jsonschema-form/blob/ff66b13dd0e74a36394f3e45e546137b024d709e/src/utils.js#L890

changes `` `${name}.${i}` `` to `` `${name}${i}.` ``.

Now that I think about it, it was honestly was kind of an arbitrary change. If you think the old style is more clear, I can change it back.

Reply:
I think you can change it to the old style, unless there are any additional advantages to this new style.